.github/workflows/reusable-create-source-tarball.yml

```diff
@@ -9,6 +9,9 @@ on:
       release:
         type: boolean
 
+permissions:
+  contents: read
+
```

Comment:
Why do some of them have an empty line, but the others don't.

Reply:
I have fixed this. 
Thank you